lib/update-fetch.js

```diff
@@ -1,27 +1,38 @@
 var path = require('path');
 var urljoin = require('url-join');
 var request = require('request');
-var buildServerRoot = 'https://builds.tessel.io/t2';
-var firmwareBuildURL = urljoin(buildServerRoot, 'firmware');
-var buildsListURL = urljoin(firmwareBuildURL, 'builds.json');
-var openWRTBuild = 'openwrt.bin';
-var firmwareBuild = 'firmware.bin';
 var gunzip = require('zlib').createGunzip();
 var extract = require('tar-stream').extract();
 var logs = require('./logs');
 var streamToBuffer = require('stream-to-buffer');
+
+
+var buildServerRoot = 'https://builds.tessel.io/t2';
+var firmwareBuildURL = urljoin(buildServerRoot, 'firmware');
+// THIS IS ONLY HERE FOR DEMONSTRATION, THE URL WILL CHANGE ONCE
+// THE BUILD DATA FORMAT IS UPDATED.
+var buildsListURL = 'https://dl.dropboxusercontent.com/u/3531958/builds.json';
```

Comment:
I've updated the `build.json` on the build server. It will take ~10 minutes to propogate but you can test with the non-Cloudfronted URL: https://s3.amazonaws.com/builds.tessel.io/t2/firmware/builds.json.